diff --git a/public/app/core/components/Footer/Footer.tsx b/public/app/core/components/Footer/Footer.tsx
index 6273951..48ec07b 100644
--- a/public/app/core/components/Footer/Footer.tsx
+++ b/public/app/core/components/Footer/Footer.tsx
@@ -15,6 +15,7 @@ export interface FooterLink {

 export let getFooterLinks = (): FooterLink[] => {
   return [
+/*
     {
       target: '_blank',
       id: 'documentation',
@@ -36,6 +37,7 @@ export let getFooterLinks = (): FooterLink[] => {
       icon: 'comments-alt',
       url: 'https://community.grafana.com/?utm_source=grafana_footer',
     },
+*/
   ];
 };

@@ -52,7 +54,7 @@ export function getVersionLinks(hideEdition?: boolean): FooterLink[] {
   const { buildInfo, licenseInfo } = config;
   const links: FooterLink[] = [];
   const stateInfo = licenseInfo.stateInfo ? ` (${licenseInfo.stateInfo})` : '';
-
+/*
   if (!hideEdition) {
     links.push({
       target: '_blank',
@@ -84,7 +86,7 @@ export function getVersionLinks(hideEdition?: boolean): FooterLink[] {
       url: 'https://grafana.com/grafana/download?utm_source=grafana_footer',
     });
   }
-
+*/
   return links;
 }
 